### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by mew2057**
Should enable PR templating.

### CODE REVIEW

**Comment by mew2057** [comment]
This _should_ give us templating in our issues.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Summary of Changes
+
+## Unit Tests
+
+## Documentation Updates
+
```
Comment (Jainbrt) [inline .github/pull_request_template.md:6]: 1. We should also do upgrade assessment from all previous supported release.
2. We should do impact assessment on OCP vs K8

Comment (smitaraut) [reply .github/pull_request_template.md:6]: Let us add "Upgrade considerations". Let us not make the PR template heavy by adding specific like OCP vs K8 difference which in most cases wont be there.

**Comment by mew2057** [comment]
@Jainbrt @smitaraut  review and merge

**Pull request merged**